routers/web/web.go

```diff
@@ -349,6 +349,12 @@ func registerRoutes(m *web.Route) {
 		m.Get("/change-password", func(ctx *context.Context) {
 			ctx.Redirect(setting.AppSubURL + "/user/settings/account")
 		})
+		wellKnownDir := wellKnownWebDir()
+		m.Get("/*", func(ctx *context.Context) {
```

Comment:
I don't think this is right. Why not use `m.Get("/.well-know/*", `

Reply:
This code inside group `m.Group("/.well-known", func() {` and my code works (I checked). All endpoints related to `"/.well-known"` is inside that group.

Reply:
Moved `m.Get("/.well-known/*", func(ctx *context.Context) {` out of `Group`.